app/views/organisations/not_live.html.erb

```diff
@@ -22,7 +22,7 @@
     <div class="inner-block">
       <div class="description">
         <%= govspeak_to_html @organisation.summary %>
-        <% unless @organisation.closed? %>
+        <% unless @organisation.closed? || @organisation.court_or_hmcts_tribunal? %>
```

Comment:
Does this evaluate to true if `@organisation` is an open, transitioned court or HMCTS tribunal? Given the commit message and the test, that doesn't seem intentional. If it is, could you adjust the test and commit message?